packages/react-core/src/components/Label/examples/Label.md

```diff
@@ -329,3 +330,24 @@ OutlinedLabels = () => (
   </React.Fragment>
 );
 ```
+
+### Router link
+```js
+import React from 'react';
+import { Label } from '@patternfly/react-core';
+import InfoCircleIcon from '@patternfly/react-icons/dist/js/icons/info-circle-icon';
+import { Link } from '@reach/router';
```

Comment:
same comment as above

Reply:
Yes.  I saw it used in Dropdown and that works... I assumed it was in the docs?